design/badges/standards.md

```diff
@@ -1,27 +1,61 @@
-# Badge standards for less problems
+# Badge standards
 
 ## Software
 Any vector design software that can export as SVG (Scalable Vector Graphics). Adobe Illustrator and Inkscape are the most commonly-used ones.
 
 Inkscape is free, so if you don't have a Creative Cloud subscription etc., it's a good place to start.
 
-## Building your badge
-Unless you are tracing over a design or have a complex design that requires a lot of nodes, refrain from using lines to build your badge, especially if this is your first one. The easiest way to begin building is by using primitives (circles, squares, polygons).
+## Where to start?
+If this is your first badge, the easiest way to begin building is by using primitives (circles, squares, polygons).
 
-### Why should I use primitives?  
-Because I said so. 
+It is very easy to shape primitives to your desire as they have a minimal amount of points and can be very easily adjusted. Unless your badge is some complex shape, then everything can be built out of rectangles, ellipses, or triangles. Most vector softwares should also have a shape builder tool that makes combining primitives easy. If your badge is a simple shape made up of primitives, you can simply duplicate the final shape and get rid of the fill in order to create the cut line around the outside of your badge.
 
-Joking aside, it's very easy to shape primitives to your desire as they have a minimal amount of points and can be very easily adjusted. Unless your badge is some complex shape, then everything can be built out of rectangles, ellipses, or triangles. Most vector softwares should also have a shape builder tool that makes combining primitives easy. Finally, you can simply duplicate the final shape and get rid of the fill in order to easily create the cut line around the outside of your badge.
+## Material choice
+See [ordering-material.md](ordering-material.md).
+Be aware that different materials will cause engravings to show more or less clearly. Generally, lighter colors are more difficult to see engravings on (because engravings are white). Pearl acrylics also make the engravings much fainter. For an example, see the 5.0 badge (fluorescent) vs the 5.7 badge (pearl).
+
+## Colors
+In this section, colors will be represented with hex codes. Hex codes can be 3 (`#xxx`) or 6 (`#xxxxxx`) digits. For the sake of simplicity, hex codes in this document will be represented by a 3-digit code. To get the corresponding 6-digit code, simply double each digit. For example, `#F00` -> `#FF0000`
+
+Trotec Ruby will recognize any grayscale color from `#000` to `#FFF`, and will recognize `#F00` as the cut path. Any other color cannot be engraved or cut by Trotec Ruby.
+
+If it is your first attempt at making a badge, do not attempt grayscale. Instead, make sure everything is either `#F00`, for the cut path, `#FFF`, for no engraving, or `#000`, for engraving.
```

Comment:
```suggestion
If it is your first attempt at making a badge, do not attempt grayscale. Instead, make sure everything is either `#F00` (red), for the cut path, `#FFF` (white), for no engraving, or `#000` (black), for engraving.
```

Just to make it abundantly clear and require no mental parsing since this section is very important